e2e/_suites/kubernetes-autodiscover/testdata/templates/elastic-agent.yml.tmpl

```diff
@@ -7,6 +7,9 @@ metadata:
 data:
   writer.sh: |-
     #!/bin/sh
+    apt-get install jq -y
+    rm /var/lib/dpkg/lock
+    apt-get install jq -y
```

Comment:
Do we need to install jq twice? Other than that, LGTM

Reply:
I do it twice after also removing the lock in case the first try fail.